### PR TITLE
NP-2628 ztNalejImage argument added

### PR DESCRIPTION
**Pull request by Carmendelope**
### What does this PR do?
adds a new argument `ZTNalejImage` that indicates which image zt-nalej is going to be used

#### What are the relevant tickets?

- [NP-2628](https://nalej.atlassian.net/browse/NP-2628)

### CODE REVIEW

File: pkg/decorators/network/zerotier/zerotier_decorator.go [hunk 1]
```diff
@@ -34,7 +34,7 @@ import (
 
 const (
 	// Name of the Docker ZT agent image
-	ZtAgentImageName = "nalej/zt-agent:v0.4.0"
+	//ZtAgentImageName = "cdelope/zt-agent:v0.5.0"
```
Comment (juanmanuel-tirado) [inline pkg/decorators/network/zerotier/zerotier_decorator.go]: Remove if no longer used.

Comment (Carmendelope) [reply pkg/decorators/network/zerotier/zerotier_decorator.go]: Of course!!!! SORRY!

File: pkg/config/config.go [hunk 2]
```diff
@@ -173,6 +175,9 @@ func (conf *Config) Validate() derrors.Error {
 		return derrors.NewInvalidArgumentError("StorageFabricAddress must be set")
 	}
 
+	if conf.ZTNalejImage == "" {
```
Comment (dhiguero) [inline pkg/config/config.go]: Only if `networkType` is set to zero tier.

File: pkg/config/config.go [hunk 3]
```diff
@@ -196,6 +202,7 @@ func (conf *Config) Print() {
 	log.Info().Uint32("port", conf.ZTSidecarPort).Msg("ZT sidecar config")
 	log.Info().Interface("networkType", conf.NetworkType).Msg("Network type")
 	log.Info().Str("unifiedLoggingAddress", conf.UnifiedLoggingAddress).Msg("Unified Logging Slave Address")
+	log.Info().Str("ZTNalejImage", conf.ZTNalejImage).Msg("ZT-Nalej image")
```
Comment (dhiguero) [inline pkg/config/config.go]: Same, only if networkType is set to ZeroTier

File: pkg/config/config.go (hunk 2 above)
Comment (Carmendelope) [reply pkg/config/config.go]: I didn`t know the network Type meant this.

Comment (Carmendelope) [reply pkg/config/config.go]: Done

File: pkg/config/config.go (hunk 3 above)
Comment (Carmendelope) [reply pkg/config/config.go]: done

**Pull request merged**